### DISCUSSION TITLE
Welcome to dedao-dl Discussions!

### DISCUSSION THREAD

**Discussion by yann0917** [yann0917/dedao-dl#97] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lawson332-eng** [comment]
您好
（注意：生成 PDF 的时候，操作过于频繁会触发 496 NoCertificate , 因此每次生成一次PDF sleep 0~5秒, 尽管如此，还是有极大可能触发操作频繁图形验证。）
触发了频繁操作后，该怎么处理

**Reply by yann0917** [reply]
登录[网页版得到](https://www.dedao.cn/) , 会出现图形验证码，验证成功之后，执行命令行 login -q  重新登录一下就行。

**Comment by taoye1982** [comment]
你好。请问最新的v1.2.6应该如何使用？登录会提示“Error: 存在登录的用户，可以进行切换登录用户”。我试了切换和查看用户，但是都会提示这条信息。用login会报错。
![捕获](https://github.com/user-attachments/assets/058a67c0-0700-4690-9437-a3652898edcc)
